src/handlers/processors.ts

```diff
@@ -18,7 +18,7 @@ export const processors: Record<string, Handler> = {
   },
   [GithubEvent.ISSUES_REOPENED]: {
     pre: [nullHandler],
-    action: [issueReopenedCallback],
+    action: [issueReopenedBlameCallback, issueReopenedCallback],
     post: [nullHandler],
```

Comment:
@0xcodercrane Does this make sense to add in post?

Reply:
Depends on the goal of `issueReopenedBlameCallback`.  In general, the post-handler would be a light one. so I think it might be better to add it in action.